bowler/tests/tool.py

```diff
@@ -146,3 +146,10 @@ def test_process_hunks_after_only_no(self, mock_patch, mock_prompt):
         mock_prompt.side_effect = ["n", "n"]
         tool.process_hunks(target, hunks)
         mock_patch.assert_not_called()
+
+    def test_validate(self):
+        tool = BowlerTool(Query().compile(), silent=False)
+        with self.assertRaises(BadTransform):
+            tool.processed_file(
+                new_text="print 'str'", filename="foo.py", old_text="print('str')"
```

Comment:
😆

Reply:
I added another case, since print is special in the grammar too.